### DISCUSSION TITLE
Trakt database crash

### DISCUSSION THREAD

**Discussion by kosorin** [trakt/showly#657] [General]
I use Trakt mainly through this app and my last sync was on Dec 9th, 2 days before the crash. So all my data are currently saved in the app. Right now they restored data prior Nov 7th and some of my data are missing.

Just in case, if they will not be able to restore all my data, is there anything I can do to save my data? For example export? I can't see any "export" button in the app.
Or will the sync be done correctly and data will be uploaded again to the Trakt servers?

**Comment by michaldrabik** [comment]
@kosorin Right now your data is stored locally in the app on your phone so when Trakt API is finally back online you should be able to just simply run Export from the app and restore all missing items from Trakt servers.

**Reply by gaurav9372** [reply]
what happened to trakt database??

**Reply by michaldrabik** [reply]
![image](https://user-images.githubusercontent.com/11537045/208303893-8ef94b28-de41-4727-8984-56f28dcc3a55.png)
